rebasehelper/application.py

```diff
@@ -407,10 +407,11 @@ def patch_sources(self, sources):
                 OutputLogger.set_patch_output('Unapplied patches:', self.rebased_patches['unapplied'])
         OutputLogger.set_patch_output('Patches:', self.rebased_patches)
 
+    @property
```

Comment:
I didn't make it intentionally as a property. I assume that I hit some combination of keys and my IDE did this behind my back. I'm very sorry for that. It's my first case...